ui/package.json

```diff
@@ -65,7 +65,7 @@
     "css-loader": "^7.1.2",
     "html-webpack-plugin": "^5.6.4",
     "jest": "^30.0.5",
-    "jest-environment-jsdom": "30.0.5",
+    "jest-environment-jsdom": "30.1.2",
```

Comment:
There's a commit at the time of this release but no tag on github nor a version bump. Let's wait...